From 702ddbd102ce9c697ed3bae076b70b7090fc06b1 Mon Sep 17 00:00:00 2001
Message-ID: <702ddbd102ce9c697ed3bae076b70b7090fc06b1.1705854493.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1705854493.git.sharlatanus@gmail.com>
References: <cover.1705854493.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Mon, 15 Jan 2024 18:39:45 +0000
Subject: [PATCH 13/19] gnu: python-synphot: Enable more options.

* gnu/packages/astronomy.scm (python-synphot)[propagated-inputs]: Add
python-dust-extinction and python-specutils. Sort alphabetically.
[native-inputs]: Add python-semantic-version.

Change-Id: If4dbcf2765508d7c7e2b7580193367ed2d038b6f
---
 gnu/packages/astronomy.scm | 9 +++++----
 1 file changed, 5 insertions(+), 4 deletions(-)

diff --git a/gnu/packages/astronomy.scm b/gnu/packages/astronomy.scm
index 7e1dd3ed82..8263d2d954 100644
--- a/gnu/packages/astronomy.scm
+++ b/gnu/packages/astronomy.scm
@@ -3945,14 +3945,15 @@ (define-public python-synphot
                   (format port "[pytest]
 python_files = test_*.py"))))))))
     (propagated-inputs
-     (list ;; python-dust-extinction ; XXX: Not packed yet, optional.
-           ;; python-specutils       ; XXX: Not packed yet, optional.
-           python-astropy
+     (list python-astropy
+           python-dust-extinction
            python-numpy
-           python-scipy))
+           python-scipy
+           python-specutils ))
     (native-inputs
      (list python-pytest
            python-pytest-astropy
+           python-semantic-version
            python-setuptools-scm))
     (home-page "https://github.com/spacetelescope/synphot_refactor")
     (synopsis "Synthetic photometry using Astropy")
-- 
2.41.0

